# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/silo/v7.patch.sparc
# Copyright (C) 2017 - 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- silo.git/common/Makefile.vanilla	2017-12-23 10:34:40.583625829 +0000
+++ silo.git/common/Makefile	2017-12-23 10:34:59.651625131 +0000
@@ -8,7 +8,7 @@
 	$(CC) $(CFLAGS) -c $*.c
 
 .S.o:
-	$(CC) $(CFLAGS) -c $*.S
+	$(CC) $(CFLAGS) -mcpu=v8 -c $*.S
 
 OBJS = sdiv.o rem.o udiv.o urem.o jmp.o printf.o console.o prom.o tree.o stringops2.o\
 	stringops1.o ffs.o divdi3.o udivdi3.o
--- silo.git/second/Makefile.vanilla	2017-12-23 10:35:36.947623766 +0000
+++ silo.git/second/Makefile	2017-12-23 10:35:55.527623086 +0000
@@ -30,7 +30,7 @@
 	$(CC) $(CFLAGS) -c $*.c -o $@
 
 .S.o:
-	$(CC) $(CFLAGS) -c $*.S -o $@
+	$(CC) $(CFLAGS) -mcpu=v8 -c $*.S -o $@
 
 # The ordering here is very significant. Please add new object files to OBJS5 only.
 OBJS1 = crt0.o
